resources/bootstrap-manifests/bootstrap-apiserver.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     - --bind-address=0.0.0.0
     - --client-ca-file=/etc/kubernetes/secrets/ca.crt
     - --cloud-provider=${cloud_provider}
-    - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Priority
+    - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultTolerationSeconds,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Priority,PodSecurityPolicy
```

Comment:
Aren't this manifest generated by bootkube? Shouldn't we change bootkube to autogenerate this (when running with an option or something)? Or at least create a bug to do this soon (and still have the manual modification)?

Reply:
Answered here: https://github.com/kinvolk/terraform-render-bootkube/pull/16#issuecomment-509504834

tl; dr it is fine, bootkube are an initial creation of assets, we and typhoon modify them here. So, fine to do modification here :)